environment.yml

```diff
@@ -1,4 +1,8 @@
----
+channels:
```

Comment:
please include just these:
```
  - nodefaults
  - bioconda
  - conda-forge
```
The `defaults` and `anaconda` channels are not free for commercial use, and we don't actually need them.

Reply:
Updated.